com.atlauncher.ATLauncher.yml

```diff
@@ -0,0 +1,55 @@
+# Credits to github/nickavem for the script, Doomsdayrs for edits
+
+app-id: com.atlauncher.ATLauncher
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.openjdk8
+command: atlauncher.sh
+build-options:
+    env:
+        PATH: /app/jre/bin:/usr/bin
+finish-args:
+    - --env=PATH=/app/jre/bin:/usr/bin:/app/bin
+    - --socket=x11
+    - --share=ipc
+    - --device=all
+    - --socket=pulseaudio
+    - --share=network
+modules:
+    - name: openjdk
+      buildsystem: simple
+      build-commands:
+          - /usr/lib/sdk/openjdk8/install.sh
+
+    - name: atlauncher
+      buildsystem: simple
+      build-commands:
+          - install -D ATLauncher-3.4.10.15.jar /app/bin/ATLauncher.jar
+          - unzip -p /app/bin/ATLauncher.jar assets/image/icon.png > ./icon.png
+          - install -D ./icon.png /app/share/icons/hicolor/256x256/apps/com.atlauncher.ATLauncher.png
+      sources:
```

Comment:
It should use https://github.com/flathub/flatpak-external-data-checker to detect updates.

Reply:
This would require x-data-checker type git.

I do not understand regex tho

Reply:
There's plenty of examples in various GitHub repositories: https://github.com/search?q=org%3Aflathub+x-checker-data&type=code

Reply:
```
$ flatpak run org.flathub.flatpak-external-data-checker com.atlauncher.ATLauncher.yml
INFO    src.checker: Checking 1 external data items
INFO    src.checker: Skipped check [1/1] file ATLauncher.jar (from com.atlauncher.ATLauncher.yml)
INFO    src.main: Check finished with 0 error(s)
```

Is this what it should look like?

Reply:
I mean, you are looking for new releases of GNOME Cheese, so probably not what you wanted? An example usage with GitHub is https://github.com/flathub/nz.mega.MEGAsync/blob/03dbeadeb29c83524598405f64df2daa15d017cb/nz.mega.MEGAsync.yml#L105, and I think there's an example in f-e-d-c README too.

Reply:
That should do it.